include/cpr/cookies.h

```diff
@@ -25,26 +51,29 @@ class Cookies {
     bool encode{true};
 
     // NOLINTNEXTLINE(google-explicit-constructor, hicpp-explicit-conversions)
-    Cookies(bool p_encode = true) : encode(p_encode) {}
-    Cookies(const std::initializer_list<std::pair<const std::string, std::string>>& pairs, bool p_encode = true) : encode(p_encode), map_{pairs} {}
+    Cookies(bool p_encode = true) : encode(p_encode){};
```

Comment:
```suggestion
    Cookies(bool p_encode = true) : encode(p_encode) {};
```

Reply:
I would like to modify the initialization like that: 
```suggestion
    Cookies(bool p_encode = true) : encode{p_encode} {};
```